test/performance/README.md

```diff
@@ -81,3 +81,18 @@ and upper bound to show. For example:
 ```
 gnuplot -c test/performance/latency-and-thpt-plot.plg data.csv 0.005 480 520
 ```
+## Profiling
+Most eventing binaries under `cmd` package are bootstrapped by either `sharedmain.Main`
+in `knative.dev/pkg/injection/sharedmain` or `adapter.Main` in `knative.dev/eventing/pkg/adapter`.
+These `Main` helper functions uses the [profiling](https://github.com/knative/pkg/blob/master/profiling/server.go)
+package to enable golang profiling by reading the `profiling.enable` flag in
+the `config-observability` configmap.
+
+To enable profiling,
+1. Add or modify `profiling.enable:true` in `config/config-observability.yaml`'s
```

Comment:
We should also add this flag to _example. I recommend copying the example from serving: https://github.com/knative/serving/blob/master/config/core/configmaps/observability.yaml#L108-L112.

Reply:
I remember when I added it to `_example` it didn't work. It has to be its own entry in the configmap I believe.

Reply:
Everything under _example is ignored, this just helps the configmap to be self-documenting.

Reply:
Oh I see. Added.